src/core/etl/src/Flow/ETL/Function/DOMElementAttribute.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Function;
+
+use Flow\ETL\Row;
+
+final class DOMElementAttribute extends ScalarFunctionChain
```

Comment:
another good scalar function would be DOMElementHasAttribute which could be user together with `when()` expression. like `when(ref('node')->domElementHasAttribute("id"))->then(...)`, but it's out of the scope of that PR.